spec/converters/json_spec.rb

```diff
@@ -0,0 +1,154 @@
+require 'spec_helper'
+require 'ronin/masscan/converters/json'
+require 'tempfile'
+
+RSpec.describe Ronin::Masscan::Converters::JSON do
+  let(:fixtures_path) { File.expand_path(File.join(__dir__, '..', 'fixtures')) }
+  let(:masscan_path)  { File.join(fixtures_path, 'converter', 'masscan.json') }
+  let(:masscan_file)  { Masscan::OutputFile.new(masscan_path) }
+  let(:ip_addr)       { IPAddr.new('93.184.216.34/32') }
+  let(:expected_status) do
+    {
+      status: :open,
+      protocol: :tcp,
+      port: 80,
+      reason: [:syn, :ack],
+      ttl: 54,
+      ip: ip_addr,
+      timestamp: Time.at(1629960621)
+    }
+  end
+
+  describe '.convert' do
+    it 'must convert masscan file to json and write it into an output' do
+      Tempfile.create do |output_file|
+        subject.masscan_file_to_json(masscan_file, output_file)
+        output_file.rewind
+
+        expect(output_file.read).to eq(JSON.dump([expected_status]))
```

Comment:
I think we should test the `.convert` method with multiple values, not just a single status value. Maybe `Masscan::Status` and `Masscan::Banner` values?